src/payment/strategies/afterpay/afterpay-payment-strategy.ts

```diff
@@ -72,7 +72,10 @@ export default class AfterpayPaymentStrategy implements PaymentStrategy {
             this._paymentMethodActionCreator.loadPaymentMethod(paymentId, options)
         );
 
-        await this._redirectToAfterpay(storeCountryName, state.paymentMethods.getPaymentMethod(paymentId));
+        const checkout = state.checkout.getCheckout();
+        const amount = checkout?.outstandingBalance || 0;
```

Comment:
just to make sure, this balance also considers store credit, right?

Reply:
yes it works, I will upload a video showing both